### PR TITLE
Update Microsoft.UI.Xaml.2.8

### PR DESCRIPTION
**Pull request by Trenly**
cc @mdanish-kh
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/120909)

### CODE REVIEW

File: manifests/m/Microsoft/UI/Xaml/2/8/8.2306.22001.0/Microsoft.UI.Xaml.2.8.installer.yaml [hunk 1]
```diff
@@ -2,15 +2,10 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
 
 PackageIdentifier: Microsoft.UI.Xaml.2.8
-PackageVersion: 2.8.5
+PackageVersion: 8.2306.22001.0
 InstallerType: appx
 UpgradeBehavior: install
 MinimumOSVersion: 10.0.15063.0
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/UI/Xaml/2/8/8.2306.22001.0/Microsoft.UI.Xaml.2.8.installer.yaml:8]:
```suggestion
MinimumOSVersion: 10.0.17763.0
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/m/Microsoft/UI/Xaml/2/8/8.2306.22001.0/Microsoft.UI.Xaml.2.8.installer.yaml (hunk 1 above)
Comment (mdanish-kh) [reply manifests/m/Microsoft/UI/Xaml/2/8/8.2306.22001.0/Microsoft.UI.Xaml.2.8.installer.yaml:8]: From AppxManifest.xml

```xml
  <Dependencies>
    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.17763.0" MaxVersionTested="10.0.18362.0" />
  </Dependencies>
```

**Review by mdanish-kh — APPROVED** [review]

**Review by mdanish-kh — APPROVED** [review]
The bot is failing to catch up

**Comment by Trenly** [comment]
@stephengillie - Can this be manually merged since it isn't removing the dependency?

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
The dependency error is incorrect:
>  Manifest Error: Package has a single manifest and is a dependency of other manifests. [PackageIdentifier.PackageVersion] Value: Microsoft.WindowsTerminal.1.18.2681.0

Does it not allow any changes?

**Comment by Trenly** [comment]
> The dependency error is incorrect:
> 
> > Manifest Error: Package has a single manifest and is a dependency of other manifests. [PackageIdentifier.PackageVersion] Value: Microsoft.WindowsTerminal.1.18.2681.0
> 
> Does it not allow any changes?

I think it detects it as a removal because the version is being changed and this is the only version in the repo

**Pull request merged**